scipy/stats/stats.py

```diff
@@ -4363,10 +4363,8 @@ def ks_2samp(data1, data2):
     data2 = np.sort(data2)
     n1 = data1.shape[0]
     n2 = data2.shape[0]
-    data_all = np.concatenate([data1, data2])
-    cdf1 = np.searchsorted(data1, data_all, side='right') / (1.0*n1)
-    cdf2 = np.searchsorted(data2, data_all, side='right') / (1.0*n2)
-    d = np.max(np.absolute(cdf1 - cdf2))
+    from . import _stats
```

Comment:
That is a good idea (e.g. the von mises could also be moved there later on), because the generated c code contains a lot of boilerplate that is needlessly repeated in each `.so` generated.